WBooks/Library/Cell/LibraryTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LibraryTableViewCell.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 12/08/21.
+//
+
+import UIKit
+
+class LibraryTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var containerView: UIView!{
+        didSet{
+            containerView.layer.cornerRadius = 10
+            containerView.layer.shadowColor = UIColor.darkGray.cgColor
+            containerView.layer.shadowOffset = CGSize(width: 0.0, height: 0.1)
+            containerView.layer.shadowOpacity = 0.2
+            containerView.layer.shadowRadius = 3
+            containerView.layer.masksToBounds = false
+            
+        }
+    }
+    @IBOutlet weak var bookImage: UIImageView!{
+        didSet{
+            bookImage.contentMode = .scaleAspectFill
+        }
+    }
+    @IBOutlet weak var titleLabel: UILabel!{
+        didSet{
+            titleLabel.font = UIFont(name: "Gill Sans", size: 16)
```

Comment:
we usually create a UIFont extension and add getters for the fonts we are going to use

Reply:
Done